text/0034-telemetry-policies.md

```diff
@@ -417,11 +417,11 @@ The `User-Agent` header is tracked on the events listed below. Our official SDKs
 
 Each endpoint API tracked sends the `User-Agent` as a `user_agent` event property as an array. If several values are contained in the `User-Agent` header, they are split by the `;` character.
 
-##### `x-meilisearch-client` Query Parameter
```

Comment:
Disclaimer, I like the name `X-MEILISEARCH-CLIENT`.

But it is not mandatory to have the `X-` prefix https://stackoverflow.com/a/3561399/2649707

Reply:
I think it's safe to say that `X-MEILISEARCH-CLIENT` will not become a standard.

I like the idea of `X-` because like most people I don't know the RFC in every detail and I know right away that it is often a custom header introduced by the API used.

Nevertheless, this is only my opinion, and let technical folks decide with a vote.

👍: To keep X-
👎  To remove it

Reply:
WHY IS IT WRITTEN IS CAPS LOCK THOUGH?
SHOULDN'T IT BE SOMETHING LIKE `X-Meilisearch-Client`? :joy:

Reply:
I DON'T KNOW 🥲, I THINK YOU ARE RIGHT